### PR TITLE
[Better Tablet Products] The informational bubble about the write with ai button is oddly placed

### LINKED ISSUE #11017: [Better Tablet Products] The informational bubble about the “write with AI” button is oddly placed
pfoUAQ-iv-p2#comment-260

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #11017

### Description

The PR 
* Sets the width of the AI toolbar to the parent view to make it look ok on 2-pane mode
* Show the tooltip only when the AI button amove the bottom of a screen on 230DP

### Testing instructions

* apply patch to show tooltip all the time
[Change_for_testing_toolbar.patch](https://github.com/woocommerce/woocommerce-android/files/14757427/Change_for_testing_toolbar.patch)
* Using tablets and phone, notice that the toolbar is shown when it's possible to show that

### Images/gif

Before

image

https://github.com/woocommerce/woocommerce-android/assets/4923871/2bc9acdc-9c16-4849-881b-72e475e189f3

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/WCProductPropertyButtonView.kt [hunk 1]
```diff
@@ -45,33 +45,94 @@ class WCProductPropertyButtonView @JvmOverloads constructor(
...
+
+    private fun showPopupWindowWhenEnoughSpace(
+        scrollableView: NestedScrollView,
+        tooltip: ProductProperty.Button.Tooltip
+    ) {
+        if (isEnoughSpaceToShowTooltip()) {
+            showTooltip(scrollableView.measuredWidth, tooltip)
+            return
         }
+        scrollableView.setOnScrollChangeListener(
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/WCProductPropertyButtonView.kt:82]: I had to use nested scroll view as we have `RecyclerView` inside of `RecyclerView` and then inside of `NestedScrollView`. I think the most nested RV doesn't do the actual views recycling, does it? @0nko that's the old code but maybe something to keep in mind

Comment (0nko) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/WCProductPropertyButtonView.kt:82]: The nested RV was to just dynamically render the product detail properties based on a model. The number of items was small so there was no need to recycle anything. It's been a while, though..

Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/WCProductPropertyButtonView.kt:82]: > The number of items was small so there was no need to recycle anything

Then it's not clear why to have an RV at all, right? Especially RV inside of RV =)

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/203">18.0</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by hafizrahman — APPROVED** [review]
the change works and looks good to me 👍🏼  Feel free to merge after resolving conflict.

**Pull request merged**